src/app.config.js

```diff
@@ -18,6 +22,13 @@ const Config = {
   clickDebounceTime: 180, // ms
   dashboardRefreshInterval: 15000, // 15 seconds
   telemetryRefreshInterval: 1000, // 1 seconds
+  simulationId: '1',
+  validExtensions: validExtensions.join(),
+  isValidExtension: (file) => {
```

Comment:
The valid extensions should be a member in the config and the helper method should be in the utilities folder in the methods.js file.